### ISSUE TITLE
Dark mode support

### ISSUE THREAD

**Issue by ericlinagora** [linagora/twake-drive-legacy#697]
This is intended to be a long running low priority issue to tag commits that make gradual improvement of dark mode support, and to list remaining unsupported UI sections

**Comment by ericlinagora** [comment]
![image](https://github.com/user-attachments/assets/6bfdae83-76d4-4f6b-a60d-2d76429792b9)

**Comment by zaheer-Khan3260** [comment]
I Want to work on this please assign to me.

**Comment by ericlinagora** [comment]
Alright, thanks zaheer-Khan3260. This task however is more of a long running one, there isn't even a list of all the places that don't have dark mode support (and some, like the accounts page, are in the webdav-express branch for organisational reasons).

**Comment by zaheer-Khan3260** [comment]
@ericlinagora Thanks to assign it to me. I am working on it.

**Comment by zaheer-Khan3260** [comment]
Hello @ericlinagora  i am facing issue while set this project locally you provide the all the instruction but after follow all the step the backend server through error of db not connected but as you mantion i already setup my docker images and all the evironment variables that you suggest but it's not working can you help me for it.

**Comment by ericlinagora** [comment]
This is irrelevant to this issue, can you please open a new one with what you tried, which docker/compose etc commands you ran, what you got/error messages etc please ?

**Comment by zaheer-Khan3260** [comment]
Yeah sure @ericlinagora , I open a new issue related to this bug.
